docs/feature-flags.md

```diff
@@ -0,0 +1,27 @@
+# Feature flags
```

Comment:
Mind linking to this in `/docs/README.md` under `Development`? That is basically the ToC for the docs there.

Thanks!

Reply:
Done